modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,115 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-0_{context}"]
+= Release notes for {gitops-title} 1.12.0
+
+{gitops-title} 1.12.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-12.0_{context}"]
+== Errata updates
+
+[id="gitops-1-12-0-security-update-advisory_{context}"]
+=== RHSA-xxxx:nnnn - {gitops-title} 1.12.0 security update advisory
+
+Issued: 2024-03-14
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2023:6220[RHSA-xxxx:nnnn]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+*The Title and errata number will change after the errata state moves from PUSH STATE to SHIPPED LIVE. The current entry is simply a placeholder at the moment*.
+
+[id="new-features-1-12-0_{context}"]
+== New features
+
+* With this update, the Argo CD CLI is supported and accessible as a productized and easily-installable RPM. The Argo CD CLI tool enables users to manage {gitops-title} from a terminal. Using this tool, users can manage ArgoCD resources, such as `Applications`, `ApplicationSets`, `AppProjects`, `User accounts`, and `GPG keys` from a client terminal. link:https://issues.redhat.com/browse/GITOPS-3389[GITOPS-3389]
+
+[NOTE]
+====
+The `argocd` executable binary file is included in the archives and the RPMs.
+====
+
+[WARNING]
+==== 	
+The Argo CD CLI tool is a Technology Preview feature.
+====
+
+* With this update, users can configure notifications in a dedicated `NotificationsConfiguration` custom resource (CR). This configuration replaces the workflow in earlier releases of {gitops-title}. To configure notifications in earlier releases, you had to edit the `argocd-notifications-cm` ConfigMap manually. Users can now set `Triggers`, `Templates`, `Services`, and `Subscriptions` as fields in the new `NotificationsConfiguration` custom resource (CR), which is managed by the {gitops-title} Operator. link:https://issues.redhat.com/browse/GITOPS-4130[GITOPS-4130]
```

Comment:
This release note takes a reverse approach to the before:now format. 

See https://redhat-documentation.github.io/supplementary-style-guide/#_bug_fix

Reply:
This is an important feature in the release and I want to make sure that the users understand it, so I have written it in this format. Could you let me know how to improve this with the approach you are suggesting?

Reply:
Fixed.